Integrations/HelloWorld/HelloWorld.yml

```diff
@@ -0,0 +1,55 @@
+category: Authentication
+commonfields:
+  id: HelloWorld
+  version: -1
+configuration:
+- display: Fetch incidents
```

Comment:
order of the parameters should be:
* server url
* api token
* is fetch
* incident type
* insecure
* proxy